gradle-conjure/src/main/java/com/palantir/gradle/conjure/CompileIrTask.java

```diff
@@ -46,25 +49,27 @@ public final File getOutputFile() {
     }
 
     @InputDirectory
+    @PathSensitive(PathSensitivity.RELATIVE)
     public final File getInputDirectory() {
         return inputDirectory.get();
     }
 
-    public final void setExecutablePath(Supplier<File> executablePath) {
-        this.executablePath = executablePath;
+    public final void setExecutableDir(Supplier<File> executableDir) {
+        this.executableDir = executableDir;
     }
 
-    @InputFile
-    public final File getExecutablePath() {
-        return executablePath.get();
+    @InputDirectory
+    @PathSensitive(PathSensitivity.RELATIVE)
+    public final File getExecutableDir() {
+        return executableDir.get();
     }
 
     @TaskAction
     public final void generate() {
         getProject().exec(execSpec -> {
             ImmutableList.Builder<String> commandArgsBuilder = ImmutableList.builder();
             commandArgsBuilder.add(
-                    executablePath.get().getAbsolutePath(),
+                    executableDir.get().toPath().resolve("bin").resolve("conjure").toFile().getAbsolutePath(),
```

Comment:
Odd to switch from File->Path->File.`toAbsolutePath().toString()` works, or `new File(executableDir.get(), "bin/conjure").getAbsolutePath()` (even on windows with modern javas!)